aws/canonical_policy.go

```diff
@@ -321,7 +321,7 @@ func uniqueStrings(arr []string) []string {
 // and it must be unescaped beofre converting to canonical form
 func unescape(ctx context.Context, d *transform.TransformData) (interface{}, error) {
 	logger := plugin.Logger(ctx)
-	logger.Trace("unescape")
+	logger.Debug("unescape")
```

Comment:
Do we still need this? It's too minimal to be useful even if we have `Debug` level turned on.

Same comment for the updated log message below too.

Reply:
Removed these debug statements.